### DISCUSSION TITLE
Install a library

### DISCUSSION THREAD

**Discussion by laisagrassi** [OpenModelica/OpenModelica#12760] [Q&A] [answered]
Hi! I am trying to install the ThermalSeparation library in OMEdit, but it does not appear in the 'install libraries' box. However, when I go to the 'index.json' file, the ThermalSeparation library is listed there, meaning it is available to be installed, but it does not appear as an option to install within OMEdit. I have even updated the software, but the problem persists. What can I do?
![WhatsApp Image 2024-07-30 at 15 33 18](https://github.com/user-attachments/assets/8dacd9b8-01d4-4931-bf53-d7073e3c59b0)
![WhatsApp Image 2024-07-30 at 15 32 40](https://github.com/user-attachments/assets/04137e54-64ab-4f3e-b0e8-1cabbbef179f)

**Comment by casella** [comment, marked as answer]
Click on the "None" checkbox in the level of support selection.
